### DISCUSSION TITLE
Default link-title for a zettel

### DISCUSSION THREAD

**Discussion by ouuan** [srid/neuron#640] [Ideas]
Now the title of the linked zettel is used as the link text. But the title of an article is different from the link text. I usually use full titles as the title of an article, but use short titles in links. As a result, I use custom link text nearly everywhere, and when I link to the same zettel, the custom link texts are usually the same, which is a big waste of time.

It will be much better if we can set the default link-title for a zettel, so that when being linked the link-title is used as the link text.

**Proposed syntax:** In the frontmatter, add a `linkTitle` field:

```markdown
---
date: 2021-07-26T19:04
linkTitle: a short title
---

# A Full Title
```

This could be a plugin.

**Comment by srid** [comment]
This is already supported. Replace `linkTitle` with `title`. A PR to mention this in the documentation is welcome! https://neuron.zettel.page/linking

**Reply by ouuan** [reply]
It seems that the `title` attribute is displayed in the zettel page, and it's rather an alternative of `# title` instead of a way to set the default link text. Is this a bug?

**Reply by srid** [reply]
Hmm. Right. If you want the page title to use actual title (even if `title` meta exists, which is used in the link regardless), then that's a new feature to be implemented (not as plugin, but perhaps as a default behaviour).

EDIT: PR on this welcome.
